<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_neko_ears">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>50</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Neko Ears</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>440</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Shot!</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_sunglass">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>80</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sunglass</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Mask</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>120</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Filter</string>
    </property>
    <widget class="QCheckBox" name="check_cartoon">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Cartoon</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="check_invert">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Invert</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="check_sketch">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Sketch</string>
     </property>
    </widget>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>label</zorder>
   <zorder>check_neko_ears</zorder>
   <zorder>pushButton</zorder>
   <zorder>check_sunglass</zorder>
   <zorder>groupBox_2</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
